woodwork/tests/utils/test_read_file.py

```diff
@@ -195,3 +196,61 @@ def test_read_file(
         schema_df = schema_df.head(kwargs["nrows"])
 
     pd.testing.assert_frame_equal(df, schema_df)
+
+
+def test_replace_nan_empty_strings(tmpdir):
+    data = {
+        "double": ["<NA>", "6.2", "4.2", "3.11"],
+        "integer": ["<NA>", "6", "4", "3"],
+        "null": ["<NA>", "", "nan", None],
+        "null_string": pd.Series(["<NA>", "", "nan", ""], dtype="string"),
+        "Int64": pd.Series([1, 2, 3, 4], dtype="Int64"),
+        "Float64": pd.Series([1.1, 2.2, 3.3, 4.4], dtype="Float64"),
+        "boolean": pd.Series([True, True, False, False], dtype="boolean"),
+        "int64": pd.Series([1, 2, 3, 4], dtype="int64"),
+        "double2": pd.Series([1, 2, 3, 4.5], dtype="float64"),
+        "bool": pd.Series([True, True, False, False], dtype="bool"),
+        "category": pd.Series([1, 2, 2, 1], dtype="category"),
+    }
+
+    expected_null_count = {
+        "double": 1,
+        "integer": 1,
+        "null": 4,
+        "null_string": 4,
+        "Int64": 0,
+        "Float64": 0,
+        "boolean": 0,
+        "int64": 0,
+        "double2": 0,
+        "bool": 0,
+        "category": 0,
+    }
+
+    df = pd.DataFrame(data=data)
+    replaced_df = replace_nan_empty_strings(df)
+    for col in replaced_df:
+        print(col)
```

Comment:
Remove?
```suggestion
```

Reply:
Yep - have this removed locally, just haven't pushed yet.